corehq/apps/app_manager/static/app_manager/js/shadow-module-settings.js

```diff
@@ -9,9 +9,15 @@ hqDefine('app_manager/js/shadow-module-settings.js', function () {
             var self = this;
             self.modules = ko.observableArray();
             self.selectedModuleId = ko.observable(selectedModuleId);
-            self.selectedModule = ko.computed(function () {
+            self.selectedModule = ko.pureComputed(function () {
```

Comment:
Using `pureComputed` (Knockout 3.2+) so that `self.selectedModule` is defined when we want to use it in `self.excludedFormIds`.